client/reader/following-manage/feed-search-results.jsx

```diff
@@ -22,14 +23,25 @@ const FollowingManageSearchFeedsResults = ( {
 	fetchNextPage,
 	forceRefresh,
 	searchResultsCount,
+	query,
 } ) => {
+	const isEmpty = !! ( query && searchResults && searchResults.length === 0 );
+	const classNames = classnames( 'following-manage__search-results', {
+		'is-empty': !! isEmpty,
+	} );
+
 	if ( ! searchResults ) {
 		return null; // todo: add placeholder
-	} else if ( searchResults.length === 0 ) {
+	} else if ( isEmpty ) {
 		return (
-			<p>
-				{ translate( 'There were no site results for your query.' ) }
-			</p>
+			<div className={ classNames }>
+				<p className="following-manage__search-results-empty-message">
+					{ translate( 'Sorry, no sites match {{italic}}%s.{{/italic}}', {
+						components: { italic: <i /> },
```

Comment:
I had exactly the same feedback, but discovered `<i />` now has semantic value in HTML5 and is probably the most appropriate element in that case.